### DISCUSSION TITLE
How to change local repositories from within lazygit?

### DISCUSSION THREAD

**Discussion by dei-layborer** [jesseduffield/lazygit#3979] [Q&A]
I'm trying to figure out how to change to a different local repository without having to quit lazygit, change directories, and then run it again.  I saw there's an option for "recent" repositories or whatever, but that doesn't help if the one I want to change to hasn't previously been opened.

**Comment by mark2185** [comment]
Yeah, you have to open it at least once in a repository for it to be added to the "recent" list.
